Bottom: 48f35e9cacd9881b12fdf1c2216f071e99167436
Top:    eb35bccb6ed76896fb86cd18bdc7ef9d7c235193
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-29 23:22:57 +0100

Refresh of hid-gtk-remove-unused-button-p

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 46a9347..4d9fdbe 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1928,8 +1928,7 @@ N_("Bring up the popup menu specified by @code{MenuName}.\n"
 
 This just pops up the specified menu.  The menu must have been defined
 as a named subresource of the Popups resource in the menu resource
-file.  If called as a response to a mouse button click, the mouse 
-button number must be specified as the second argument.  
+file.
 
 %end-doc */
 
@@ -1938,16 +1937,10 @@ static int
 Popup (int argc, char **argv, Coord x, Coord y)
 {
   GtkMenu *menu;
-  guint button;
 
   if (argc != 1 && argc != 2)
     AFAIL (popup);
 
-  if (argc == 1)
-    button = 0;
-  else
-    button = atoi (argv[1]);
-
   menu = ghid_main_menu_get_popup (GHID_MAIN_MENU (ghidgui->menu_bar), argv[0]);
   if (! GTK_IS_MENU (menu))
     {
